### PR TITLE
tests: Positive test for setting unused dynamic state

### PR DESCRIPTION
**Pull request by ziga-lunarg**
Test that binding a graphics pipeline width dynamic line width state does not affect a ray tracing pipeline

### CODE REVIEW

File: tests/unit/ray_tracing_positive.cpp [hunk 1]
```diff
@@ -976,4 +977,88 @@ TEST_F(PositiveRayTracing, UpdatedFirstVertex) {
...
+    pipeline.AddGlslClosestHitShader(closest_hit);
+
+    pipeline.AddBinding(VK_DESCRIPTOR_TYPE_ACCELERATION_STRUCTURE_KHR, 0);
+    pipeline.CreateDescriptorSet();
+    vkt::as::BuildGeometryInfoKHR tlas(vkt::as::blueprint::BuildOnDeviceTopLevel(*m_device, *m_default_queue, *m_commandBuffer));
+    pipeline.GetDescriptorSet().WriteDescriptorAccelStruct(0, 1, &tlas.GetDstAS()->handle());
+    pipeline.GetDescriptorSet().UpdateDescriptorSets();
+
+    // Build pipeline
+    pipeline.Build();
```
Comment (spencer-lunarg) [inline tests/unit/ray_tracing_positive.cpp:1044]: this RTX pipeline is missing `VK_DYNAMIC_STATE_RAY_TRACING_PIPELINE_STACK_SIZE_KHR`

Comment (spencer-lunarg) [reply tests/unit/ray_tracing_positive.cpp:1044]: I think you can just go

```
    pipeline.AddDynamicState(VK_DYNAMIC_STATE_RAY_TRACING_PIPELINE_STACK_SIZE_KHR);
 ```

Comment (ziga-lunarg) [reply tests/unit/ray_tracing_positive.cpp:1044]: Is `VK_DYNAMIC_STATE_RAY_TRACING_PIPELINE_STACK_SIZE_KHR` relevant here? The github issue only says that it is the only dynamic state allowed for ray tracing pipelines, not that it is or should be set.

Comment (spencer-lunarg) [reply tests/unit/ray_tracing_positive.cpp:1044]: you are right, took a close look.

**Review by spencer-lunarg — APPROVED** [review]
LGTM - Thanks for generating a test faster than ChatGPT could

**Pull request merged**